tests/base.py

```diff
@@ -296,14 +298,16 @@ class SgTestConfig(object):
     def __init__(self):
 
         for key in self.config_keys():
-
             # Look for any environment variables that match our test
             # configuration naming of "SG_{KEY}". Default is None.
             value = os.environ.get('SG_%s' % (str(key).upper()))
             if key in ['mock']:
                 value = (value == None) or (str(value).lower() in ['true','1'])
             setattr(self, key, value)
 
+        if not getattr(self, "human_password", None):
+            setattr(self, "human_password", os.urandom(16).encode('hex'))
```

Comment:
why doing the check for if `human_password` is already set? Is it necessary? Will that case work? I suggest either remote it or explain in a comment.